scripts/d2uChangelog.js

```diff
@@ -0,0 +1,2 @@
+const dos2unix = require('ssp-dos2unix').dos2unix
+let converted = dos2unix('CHANGELOG.md', {feedback: true, writable: true})
```

Comment:
No need to do this right now.
Do we want to handle possible errors during the conversion?
(Personally, I don't think we should care too much and not even store the output 😝 )